docs/libraries/nodejs/examples.md

```diff
@@ -180,62 +180,62 @@ Example of output:
+It starts generating addresses for the provided `seed` and `.accountIndex` from `.initialAddressIndex(index)`, and checks for a balance of each of the generated addresses. Since it does not know how many addresses are used in fact, there is a condition set by the `.gapLimit(amount)` argument to know when to stop searching. If the `.gapLimit` amount of addresses in a row have no balance, the function returns results and searching does not continue.
 
 ## Messages, payload and transactions
 
-Before we continue, let's introduce some additional terms that describe an unit that is actually broadcasted in IOTA network. IOTA is based on a concept of `messages` and `payloads`.
+Before we continue, let's introduce some additional terms that describe a unit that is actually broadcasted in the IOTA network. IOTA is based on a concept of `messages` and `payloads`.
 
-`Message` is a data structure that is actually being broadcasted in IOTA network and represent a node (vertex) in the Tangle graph. It can refer to up to 8 previous messages and once a message was attached to the Tangle and approved by a milestone, the Tangle structure ensures the content of the message is unaltered. Every message is referenced by `message_id` which is based on a hash algorithm of binary content of the message. `Message` is an atomic unit that is confirmed by network as a whole.
+`Message` is a data structure that is actually being broadcasted in the IOTA network and represents a node (vertex) in the Tangle graph. It can refer to up to 8 previous messages and once a message is attached to the Tangle and approved by a milestone, the Tangle structure ensures the content of the message is unaltered. Every message is referenced by `message_id` which is based on a hash algorithm of binary content of the message. `Message` is an atomic unit that is confirmed by the network as a whole.
 
 > IOTA is no longer based on ternary. IOTA 1.5 (Chrysalis) uses binary to encode and broadcast all underlying data entities
 
-`Message` is broadcasted using a binary format, is arbitrary size (up to 35 kB) and it can hold a variable sets of information so called `payloads`. Number of payloads a single message can encapsulate is not given (even a message without any `payload` at all is completely valid).
+`Message` is broadcasted using a binary format, is arbitrary in size (up to 35 kB), and it can hold variable sets of information so called `payloads`. The number of payloads a single message can encapsulate is not given (even a message without any `payload` at all is completely valid).
```

Comment:
```suggestion
`Message` is broadcasted using a binary format, is arbitrary in size (up to 32.8 kB), and it can hold variable sets of information so called `payloads`. The number of payloads a single message can encapsulate is not given (even a message without any `payload` at all is completely valid).
```